andela/travel_tool_front
# #159730632 Configure storybook for component development

#### What does this PR do?
This PR configures the storybook tool to make working on related components
 more efficient, easier to reuse and have better component documentation.


#### Description of Task to be completed?
- Install and configure storybook into the App
- Run storybook using Docker commands

#### How should this be manually tested?
- Clone the repository
- Install all dependencies using `yarn install`
- Ensure docker is installed, to install docker click [here](https://www.docker.com/products/docker-desktop)
- Run `make components` docker build command on the terminal to run storybook in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:9001` and you should be able to see the storybook page displayed.


#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#159730632](https://www.pivotaltracker.com/story/show/159730632)


#### Screenshots (if appropriate)

<img width="1280" alt="screen shot 2018-08-14 at 17 28 20" src="https://user-images.githubusercontent.com/12033494/44097997-ac1b95fe-9fe7-11e8-9cbe-261378783715.png">



#### Questions:
None
